src/status_im/contexts/chat/messenger/messages/view.cljs

```diff
@@ -57,6 +57,7 @@
                                 chat-screen-layout-calculations-complete?}]
     (when *screen-loaded?*
       (rn/use-mount #(reset! *screen-loaded?* true)))
+    (rn/use-unmount #(rf/dispatch [:chat/close]))
```

Comment:
in the current nav library component is unmounted later , so there will be edge cases if you close and open screen quick enough, so this won't work really well

Reply:
Hmm... yeah, that's true!, using unmount won't work for quick interactions.

I'll think about a different solution. Thanks for spotting it @flexsurfer :+1: